### PR TITLE
Typescript react boilerplate + some directory structures

### PR DESCRIPTION
**Pull request by SamZhang02**
Mainly react boilerplates + some backend directories structuring. Let me know if something doesn't look right.

### CODE REVIEW

File: client/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Getting Started with Create React App
```
Comment (terror) [inline client/README.md:1]: Don't need to keep this around

Comment (SamZhang02) [reply client/README.md]: Damn bro woke up and started reviewing code.
Gotcha, removed README.

**Comment by terror** [comment]
I think we should use poetry (https://python-poetry.org/) for managing backend dependencies, its alot nicer than a requirements.txt file + venv setup. I can set this up in another PR but its up to you if you wanna do it here.

Also we should get rid of all the junk files CRA gives us, like the logos, generated css, favicon, etc. We should only keep what we need.

**Comment by SamZhang02** [comment]
> I think we should use poetry (https://python-poetry.org/) for managing backend dependencies, its alot nicer than a requirements.txt file + venv setup. I can set this up in another PR but its up to you if you wanna do it here.

Sure, that sounds good. You can set it up in another PR while I look into it today or so.

>Also we should get rid of all the junk files CRA gives us, like the logos, generated css, favicon, etc. We should only keep what we need.

Gotcha, ill remove them later today and let you know when its done.

**Comment by SamZhang02** [comment]
I removed auto-generated logos, templates, and I cleared most contents from the main css files and ts files. 
This is my first time doing reacts tho, let me know if I may have removed anything that I was not supposed to.

**Comment by terror** [comment]
> I removed auto-generated logos, templates, and I cleared most contents from the main css files and ts files. This is my first time doing reacts tho, let me know if I may have removed anything that I was not supposed to.

I'd remove the files entirely if they're empty. Same goes for files that just have comments in them. We should only commit things that contribute actual functionality to the project.

**Comment by SamZhang02** [comment]
I have removed the majority of the files left, since they were mostly empty, just had some import statements or were mainly comments.

I have only kept a very minimal barebone of React config files: `package-lock.json`, `package.json`, etc. for now, and I will start pushing code only once they are actually functional and relevant to the product

File: data/processor.py [hunk 2]
```diff
@@ -0,0 +1 @@
+#TODO: Processor to process and store courses JSON into db
```
Comment (terror) [inline data/processor.py:1]: This should go to, will create it when we need to implement the db/json processing stuff

**Comment by terror** [comment]
Other than my previous comment, LGTM

**Comment by SamZhang02** [comment]
Deleted :)

**Review by terror — APPROVED** [review]

**Pull request merged**